core-sitemaps.php

```diff
@@ -2,13 +2,13 @@
 /**
  * Main setup.
  *
- * @package         Core_Sitemaps
+ * @package Sitemaps
```

Comment:
Not sure about updating the package names, those will be different for core anyway. We should either use the WordPress ones (`@package WordPress`, `@subpackage Sitemaps`) or leave untouched I think.

Reply:
I'll go ahead and change to use the core ones as this PR is about prepping for merge,

Reply:
Looks like this is already the case in the codebase except this instance the main plugin file. Since this file goes away anyway with the merge I am leaving as is.